support/lti.js

```diff
@@ -134,7 +134,8 @@ function getLtiOptions(options) {
     );
 
     refinedOptions.ltiLaunchUrl =
-        refinedOptions.ltiLaunchUrl || `${refinedOptions.ltiBaseLaunchUrl}${refinedOptions.ltiResourceId}`;
+        refinedOptions.ltiLaunchUrl ||
+        `${refinedOptions.ltiBaseLaunchUrl}${refinedOptions.ltiResourceId ? refinedOptions.ltiResourceId : ''}`;
```

Comment:
```suggestion
        `${refinedOptions.ltiBaseLaunchUrl}${refinedOptions.ltiResourceId || ''}`;
```
The same but shorter?

Reply:
> The same but shorter?

Indeed, I don't know why I love the ternary operator so much